src/Microsoft.DocAsCode.Build.ReferenceBase/ApplyOverwriteDocumentForReferenceBase.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Build.ReferenceBase
+{
+    using System.Collections.Generic;
+
+    using Microsoft.DocAsCode.Build.Common;
+    using Microsoft.DocAsCode.DataContracts.Common;
+    using Microsoft.DocAsCode.Plugins;
+
+    /// <summary>
+    /// Base document build step for reference to apply overwrite document.
+    /// </summary>
+    /// <typeparam name="T">The type of model that overwrite document is applied to.</typeparam>
+    public abstract class ApplyOverwriteDocumentForReferenceBase<T> : ApplyOverwriteDocument
+            where T : class, IOverwriteDocumentViewModel
+    {
+        public override int BuildOrder => 0x10;
```

Comment:
The default value 0x10 should work for most cases, and child class can also override it again if needed.

---
In reply to: [93396864](https://github.com/dotnet/docfx/pull/1030#discussion_r93396864) [](ancestors = 93396864)